jbmc/unit/java-testing-utils/load_java_class.cpp

```diff
@@ -94,13 +94,12 @@ symbol_tablet load_java_class(
   std::string filename=java_class_name + ".class";
 
   // Construct a lazy_goto_modelt
-  null_message_handlert message_handler;
   lazy_goto_modelt lazy_goto_model(
     [] (goto_model_functiont &function, const abstract_goto_modelt &model) { }, // NOLINT (*)
     [] (goto_modelt &goto_model) { return false; }, // NOLINT (*)
     [] (const irep_idt &name) { return false; },
     [] (const irep_idt &function_name, symbol_table_baset &symbol_table, goto_functiont &function, bool body_available) { return false; }, // NOLINT (*)
-    message_handler);
+    null_message_handler);
```

Comment:
I'd suggest to declare the null-message-handler once for all unit tests, because indeed it makes sense to silence output in unit tests.